src/App.tsx

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './App.css'
 import TodoList from "./TodoList.tsx";
 import AddTodoForm from "./AddTodoForm.tsx";
 
 function App() {
-
+    const [newTodo, setNewTodo] = useState<string>();
+    const onAddTodo = (title: string)=> {
```

Comment:
nit: nice that this serves as an interface to pass down to `AddTodoForm`, and could account for future refactoring. However in the context of this PR, I believe you could also just pass the setter itself as a prop